src_ext/Makefile

```
@@ -11,11 +11,11 @@ endif
 
 PATCH ?= patch
 
-URL_ocaml = https://caml.inria.fr/pub/distrib/ocaml-4.09/ocaml-4.09.1.tar.gz
-MD5_ocaml = 1a7fc32aebbad6cb821ef26a396f78e7
+URL_ocaml = http://caml.inria.fr/pub/distrib/ocaml-4.10/ocaml-4.10.0.tar.gz
```

although, given that we release the GitHub URLs to opam-repository, I think:
```suggestion
URL_ocaml = https://github.com/ocaml/ocaml/archive/4.10.0.tar.gz
```
is better, although you'll need the small change in https://github.com/dra27/opam/commit/755a760820fbd8ce331245e78a0170365cd5bca2#diff-a9c3fb4f36b2f2f2d0a5a6fc475ede42 (that change may be a good idea anyway as it makes testing a variant compiler easier)